packages/cascara/src/modules/DataTextArea/DataTextArea.js

```diff
@@ -22,8 +22,8 @@ const propTypes = {
 
 const DataTextArea = ({
   attribute,
-  isEditable = true,
-  isLabeled = true,
```

Comment:
@lordpixel I think there was a misunderstanding. We do not want to remove these default props. We should keep them. My comment was pointing out that in app_web you are explicitly defining if a module is labeled or not. We should not expose isLabeled outside of Cascara. It should be private.

That means, if someone defines isLabeled on a module object, we should throw it away.

isLabeled is being set by the table or form component. When we map over modules in a Table or Form, we should explicitly set isLabeled for our use case in Table or Form. We should not pass it down from an object config.

There is literally no use case where a Module should be labeled in a Table. Therefore, we should not force developers to keep defining that over and over.

Reply:
Changed. @brianespinosa, my reasoning for removing the default `true` in all of those props was to prevent us from unnecessarily passing the prop when is not needed.

A default `true` forces us to pass `false` when we don't actually want a label, it would be easier and less prone to error that if we don't want a label we just simply DO NOT PASS the prop at all.

Reply:
In other words: `isLabeled` prop should be `false` by default and if the module requires a label we pass `isLabeled` = `true`. Not the other way around.